src/pages/home/_trade-the-way-you-like.tsx

```diff
@@ -1,11 +1,14 @@
-import React from 'react'
+import * as React from 'react'
```

Comment:
can we use [this method](https://stackoverflow.com/a/55285991/8699388) instead of import * from react?